Sources/MintKit/CustomShell.swift

```diff
@@ -0,0 +1,59 @@
+import SwiftShell
+
+public protocol CustomShell {
+    func run(bash: String, env: [String: String], directory: String?) -> RunOutput
+    func runAndPrint(bash: String, env: [String: String], directory: String?) throws
```

Comment:
I don't think this one is necessary. It's same as the one below, isn't it?

Reply:
Thank you for your review.

Regarding `runAndPrint` there is a slight difference between the variants.
`runAndPrint(bash: "echo hello world")` will be executed as `/bin/bash -c "echo hello world"`,
whereas `runAndPrint(_:_:env:directory:)` requires a full path like `runAndPrint("/bin/bash", ["-c", "echo hello world"])`. Within Mint itself, it is used to run the installed packed.

Maybe we should rename one variant?